deploy/kubernetes/rbac.yaml

```diff
@@ -53,6 +53,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
I've been debating with myself whether this should be really enabled or listed here as a comment block (similar to secrets). I decided to enable it because the rbac.yaml then is simpler to use and the risk is lower compared to enabled access to secrets.

Reply:
Done.

Reply:
I also made this more restrictive: we only need permissions inside the namespace.